src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -50,22 +54,20 @@ public void armSpeed(double speed) {
     public boolean isInBound(double speed){
         if((getPosition() < 3) && (speed < 0)) return false;
         if ((getPosition() > 113) && (speed > 0)) return false;
-        
         return true;
     }
 
     public double getPosition(){
         return ((leftArmMotor.getEncoder().getPosition()) * 360.0) / 16.0;
     }
 
-    // public double getXAxisPosition() {
-    //     return ArmConstants.To360Scope(getPosition());
-    // }
-
     public double getVelocity(){
```

Comment:
You need to denote the units being returned here.  getVelocity() returns rpm, and you are converting it into RadiansPerMinute.

Reply:
Does velocity need to be adjusted by the gear ratio as well?  (probably)